indicator.js

```diff
@@ -0,0 +1,126 @@
+"use strict"
+
+const { GObject, Gio } = imports.gi;
+
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+
+const Logger = Me.imports.logger;
+
+function makeQuickSettings() {
+  const Main = imports.ui.main;
+  const PopupMenu = imports.ui.popupMenu;
+  const QuickSettings = imports.ui.quickSettings;
+
+  const QuickSettingsMenu = imports.ui.main.panel.statusArea.quickSettings;
+
+  const SettingsPopupSwitch = GObject.registerClass(
+  class SettingsPopupSwitch extends PopupMenu.PopupSwitchMenuItem {
+    _init(title, settings, bind) {
+      this._settings = settings;
+      const active = !!this._settings.get_boolean(bind);
+      super._init(title, active);
+      Logger.info(bind, active);
+      this.connect('toggled', item => this._settings.set_boolean(bind, item.state));
+    }
+  });
+
+  const FeatureMenuToggle = GObject.registerClass(
+  class FeatureMenuToggle extends QuickSettings.QuickMenuToggle {
+    _init(settings, extWm) {
+      const iconName = 'focus-windows-symbolic';
+      const title = _('Tiling');
```

Comment:
We can follow this more concise format for text instead of having them in `messages.js` - the only reason for the current is to find them in a single place.

Reply:
If gjs had better esm and Typescript support, then it would make more sense to me to store all the strings on an enum module, but since as it is you have to keep messages.js open in order to copy identifiers, imo i think it's better to inline everything

Reply:
Let's do it all inline